### PR TITLE
Modify webRtc stats

### PR DESCRIPTION
**Pull request by alber2510**
Modify webRtc stats
Improve performance to calculate if we should add a new sample

### CODE REVIEW

File: src/rtcModule/IRtcStats.h [hunk 1]
```diff
@@ -34,6 +35,9 @@ struct Sample
             short width = 0;        // width of frame
             short height = 0;       // height of frame
             long bwav = 0;          // bandwidth available
+            long firtx = 0;
```
Comment (sergiohs84) [inline src/rtcModule/IRtcStats.h]: Please, add some comment to this variable and the ones below

File: src/rtcModule/rtcStats.cpp [hunk 2]
```diff
@@ -351,6 +370,7 @@ void RtcStats::toJson(std::string& json) const
     JSON_SUBOBJ("samples");
         JSON_ADD_SAMPLES(, ts);
         JSON_ADD_SAMPLES(, lq);
+         JSON_ADD_SAMPLES(, f);
```
Comment (sergiohs84) [inline src/rtcModule/rtcStats.cpp]: It seems there's a spare space?

File: src/rtcModule/webrtc.cpp [hunk 3]
```diff
@@ -2651,7 +2651,7 @@ void Session::submitStats(TermCode termCode, const std::string& errInfo)
     }
 
     std::string stats = mStatRecorder->terminate(info);
-    mCall.mManager.mClient.api.sdk.sendChatStats(stats.c_str());
+    mCall.mManager.mClient.api.sdk.sendChatStats(stats.c_str(), 1380);
```
Comment (sergiohs84) [inline src/rtcModule/webrtc.cpp]: It would be better to create a `define` for the port number. What about `CHATSTATS_PORT`?

Comment (alber2510) [reply src/rtcModule/webrtc.cpp]: Solved it

File: src/rtcModule/rtcStats.cpp (hunk 2 above)
Comment (alber2510) [reply src/rtcModule/rtcStats.cpp]: Solved it

File: src/rtcModule/IRtcStats.h (hunk 1 above)
Comment (alber2510) [reply src/rtcModule/IRtcStats.h]: Solved it

**Pull request merged**